src/it/verify-it/verify.groovy

```diff
@@ -31,6 +35,34 @@ assert new File(basedir, 'target/generated-sources/localizer/org/jenkinsci/tools
 content = new File(basedir, 'target/generated-sources/localizer/org/jenkinsci/tools/hpi/its/Messages.java').text;
 assert content.contains(" holder.format(\"it.msg\");");
 
+assert new File(basedir, 'target/verify-it/META-INF/MANIFEST.MF').exists()
+
+InputStream is = Files.newInputStream(new File(basedir, 'target/verify-it/META-INF/MANIFEST.MF').toPath())
+try {
+  Manifest manifest = new Manifest(is)
+  assert !manifest.getMainAttributes().getValue('Build-Jdk-Spec').isEmpty()
+  assert manifest.getMainAttributes().getValue('Created-By').startsWith('Maven Archiver')
+  assert manifest.getMainAttributes().getValue('Extension-Name') == null // was provided by Maven 2, but core prefers Short-Name
```

Comment:
Maven 2 set this to the short name, but Maven 3 does not. Core always checks `Short-Name` first and only falls back to `Extension-Name` if `Short-Name` is not present. And we always set `Short-Name` in this plugin. So this was always redundant and can be safely removed.